src/main/java/games/strategy/engine/pbem/TripleAForumPoster.java

```diff
@@ -24,13 +24,14 @@
 import games.strategy.debug.ClientLogger;
 import games.strategy.engine.framework.system.HttpProxy;
 import games.strategy.net.OpenFileUtility;
+import games.strategy.triplea.UrlConstants;
 import games.strategy.triplea.help.HelpSupport;
 
 public class TripleAForumPoster extends AbstractForumPoster {
 
   private static final long serialVersionUID = -3380344469767981030L;
 
-  private static final String tripleAForumURL = "https://forums.triplea-game.org";
+  private static final String tripleAForumURL = UrlConstants.TRIPLEA_FORUM.toString();
```

Comment:
Just noting that `UrlConstants#TRIPLEA_FORUM` is an `http` URL.  NBD because the host is set to redirect to `https`.  I'll submit a follow-up PR to change it to `https` for consistency.